Dockerfile

```diff
@@ -1,14 +1,16 @@
 FROM alpine:3.12
 
-RUN apk --no-cache add curl git
+RUN apk --update --no-cache add curl git
```

Comment:
being very picky here is if this container's size is important that makes sense to uninstall curl and git afterwards.

Reply:
that part I don't know about. @aaron-steinfeld do you know about this? if we don't need curl or git after I agree and will uninstall and also remove from the README

Reply:
I don't believe we need to keep curl, but since this is a build image that will need to check out code, including git makes sense

Reply:
@ravisingal do you remember if there was a specific reason for curl? I can't recall one but it looks like it was included explicitly.

Reply:
yes, it was added to run the next step.